StripeFinancialConnections/StripeFinancialConnections/Source/Native/NetworkingLinkLoginWarmup/NetworkingLinkLoginWarmupViewController.swift

```diff
@@ -95,29 +107,33 @@ final class NetworkingLinkLoginWarmupViewController: SheetViewController {
     }
 
     private func didSelectSkip() {
-        dataSource.analyticsClient.log(
-            eventName: "click.skip_sign_in",
-            pane: .networkingLinkLoginWarmup
-        )
-        dataSource.disableNetworking()
-            .observe { [weak self] result in
-                guard let self = self else { return }
-                switch result {
-                case .success(let manifest):
-                    self.delegate?.networkingLinkLoginWarmupViewController(
-                        self,
-                        didSelectSkipWithManifest: manifest
-                    )
-                case .failure(let error):
-                    self.dataSource
-                        .analyticsClient
-                        .logUnexpectedError(
-                            error,
-                            errorName: "DisableNetworkingError",
-                            pane: .networkingLinkLoginWarmup
+        if dataSource.manifest.isProductInstantDebits {
+            delegate?.networkingLinkLoginWarmupViewControllerDidSelectCancel(self)
+        } else {
```

Comment:
non-blocking: any concern about a future change that accidentally didn't set the delegate when isProductInstantDebits which would then silently do nothing? 

worth a test for a nil delegate and fall through to the else case if nil? or do something to help debug if it did happen?

Reply:
Hmm, that's a good point! To be honest, if the delegate is nil here I think it's better to do nothing than to fall back on the `else` case (since that behavior is incorrect for instant debits). I'll add some logging for this scenario though!